tempesta_fw/http.c

```diff
@@ -1803,6 +1801,14 @@ tfw_http_conn_cli_drop(TfwCliConn *cli_conn)
 	list_for_each_entry_safe(req, tmp, seq_queue, msg.seq_list) {
 		req->flags |= TFW_HTTP_F_REQ_DROP;
 		list_del_init(&req->msg.seq_list);
+		/*
+		 * Response is processed and removed from fwd_queue, need to be
+		 * destroyed.
+		 */
+		if (req->resp && (req->resp->flags & TFW_HTTP_F_RESP_READY)) {
+			tfw_http_resp_pair_free(req);
+			TFW_INC_STAT_BH(serv.msgs_otherr);
```

Comment:
Why this is a server error? It seems that a server successfully respond to the request and we're here just because the client drops the connection?

Reply:
Here is output of performance statistics before the fix:
```
. . .
Server messages received		: 20
Server messages forwarded		: 16
Server messages parsing errors		: 0
Server messages filtered out		: 0
Server messages other errors		: 3
. . .
```
In this stats we see, that one response have disappeared (`received != forwarded + p_errors + filtered + o_errors)`. Other errors is the most suitable counter, which is used for the same situation in other places in code, e.g. https://github.com/tempesta-tech/tempesta/blob/43e35e9742b7f82adb1411d94a5bf7ff4d310c17/tempesta_fw/http.c#L2209-L2214